radicle-node/src/tests/e2e.rs

```diff
@@ -272,8 +271,20 @@ fn test_fetch_trusted_remotes() {
         .unwrap()
         .collect::<Result<HashSet<_>, _>>()
         .unwrap();
-    // TODO: This fails because we are fetching all namespaces at the moment.
-    assert_eq!(bob_remotes.len(), trusted.len() + 1);
+    assert_eq!(bob_remotes, Some(alice.id).into_iter().collect());
+
+    // TODO(finto): we have to fetch again to get the other trusted remotes
+    // We need to be able to thread through tracking in the fetch to
+    // look up the trusted peers -- or possibly some other method.
```

Comment:
Ok cool I'll take a look at the PR after this one is merged.  The comments confusing because of the 'thread through term', and where the 'trusted remotes' information is coming from.  As it stands its a little confusing.

Reply:
Sorry, I'll write a better comment! :) It was likely just a stream of consciousness as I thought about how to solve the problem

Reply:
All good :)